src/bin/conserve.rs

```diff
@@ -147,6 +148,16 @@ fn make_clap<'a, 'b>() -> clap::App<'a, 'b> {
                                 .about("List hashes of all blocks referenced by an index")
                                 .arg(Arg::with_name("archive").required(true)),
                         ),
+                )
+                .subcommand(
+                    SubCommand::with_name("index")
+                        .about("Debug index")
+                        .subcommand(
+                            SubCommand::with_name("dump")
+                                .about("Show the stored index for the given band")
+                                .arg(Arg::with_name("archive").required(true))
+                                .arg(Arg::with_name("band-id").required(true)),
```

Comment:
For consistency with other commands, let's use `backup_arg` rather than a positional argument.